package.json

```diff
@@ -18,50 +18,50 @@
     "prepush": "BRANCH=local npm test && npm run tslint",
     "predeploy": "npm install && npm run prepush",
     "security-checks": "git secrets --scan && git log -p | scanrepo",
-    "tslint": "tslint src/**/*.ts tests/**/*.ts -q",
+    "tslint": "tslint src/**/*.ts tests/**/*.ts -q --fix",
```

Comment:
Was this change intended? I think this will let sonar-scanner pass even if the committed code has listing issues

Reply:
Nope. Taking it out